multicluster/cmd/gateways.go

```diff
@@ -127,19 +130,34 @@ func newGatewaysCommand() *cobra.Command {
 					if !isTargetClusterMetric(metrics, gateway.clusterName) {
 						continue
 					}
-					gatewayStatus.latency = uint64(metrics.GetGauge().GetValue())
+					gatewayStatus.Latency = uint64(metrics.GetGauge().GetValue())
 					break
 				}
 
 				statuses = append(statuses, gatewayStatus)
 			}
-			renderGateways(statuses, stdout)
+
+			switch opts.output {
+			case "":
```

Comment:
I think we should maybe treat this in a similar way as we do in the viz counterpart where the output can either be "table", or "json", with the default being "table". https://github.com/linkerd/linkerd2/blob/main/viz/cmd/stat.go

Reply:
Done @mateiidavid 
Added table output in the default.